config/locales/in_person_proofing/en.yml

```diff
@@ -126,7 +133,7 @@ en:
       po_search:
         location: Find a participating Post Office
       prepare: Verify your identity in person
-      state_id: Enter the information on your ID
+      state_id_milestone_2: Enter the information on your state-issued ID
```

Comment:
We should use the original key, `state_id`. The `_milestone_2` piece isn't useful when we don't need to distinguish it from another heading